### PR TITLE
DriveBase

### PR DESCRIPTION
**Pull request by tonyn10**
DriveBaseSubsystem class: arcade drive implemented, tank drive could be an extra thing implemented later.
DriveBaseTeleopCommand class
MotorController class: organizes all the SparkMax motor controllers.

### CODE REVIEW

File: src/main/java/frc/robot/Main.java [hunk 1]
```diff
@@ -6,19 +6,18 @@
 
 import edu.wpi.first.wpilibj.RobotBase;
 
-/**
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Main.java:9]: Something wrong with block comments?

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -4,12 +4,44 @@
...
+//<p>It is advised to statically import this class (or one of its inner classes) wherever the
+//constants are needed, to reduce verbosity.
+
+public final class Constants {
+
+    // *** Values to be changed, these are placeholder values for now ***
+    
+    
+    // Constants for wheel motors
+
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Constants.java:17]: Add a TODO comment to replace motor controller IDs with real values

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -4,36 +4,32 @@
...
- * subsystems, commands, and button mappings) should be declared here.
- */
+ // This class is where the bulk of the robot should be declared. Since Command-based is a
+ // "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ // periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ // subsystems, commands, and button mappings) should be declared here.
+
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
```
Comment (azvanderpas) [inline src/main/java/frc/robot/RobotContainer.java]: Don't forget to instantiation your command and subsystem here

**Review by azvanderpas — CHANGES_REQUESTED** [review]

**Comment by tonyn10** [comment]
> DriveBaseSubsystem class: arcade drive implemented, tank drive could be an extra thing implemented later. DriveBaseTeleopCommand class MotorController class: organizes all the SparkMax motor controllers.

Fixed comment format and Robot Container. Should be good to merge to main now.

**Review by sarango2158 — APPROVED** [review]
Excellent start, very clean and easy to read!

**Review by azvanderpas — APPROVED** [review]

**Pull request merged**